Challenge_1/src/agent.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+  TransactionEvent,
+} from "forta-agent";
+
+export const CREATE_AGENT_EVENT_ABI =
+  "function createAgent(uint256 agentId,address ,string metadata,uint256[] chainIds)";
+export const UPDATE_AGENT_EVENT_ABI =
+  "function updateAgent(uint256 agentId,string metadata,uint256[] chainIds)";
+export const REGISTRY_ADDRESS = "0x61447385B019187daa48e91c55c02AF1F1f3F863";
+export const FORTA_BOTS_ADDRESS = "0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8";
+
+function provideHandleTransaction(
```

Comment:
```suggestion
export function provideHandleTransaction(
```
This suggestion to clean up how to import it to the test file.